lang/ja.js

```diff
@@ -18,10 +18,10 @@ export default {
 	"components.filter.clearDescription": "{filterName} フィルタのクリア",
 	"components.filter.clearDescriptionSingle": "フィルタのクリア",
 	"components.filter.loading": "フィルタのロード中",
-	"components.filter.filterCountDescription": "{number, plural, =0 {フィルタは適用されていません。} other {{number} 個のフィルタが適用されています。}}",
+	"components.filter.filterCountDescription": "{number, plural, =0 {フィルタは適用されていません。} one {1 個のフィルタが適用されています。} other {{number} 個のフィルタが適用されています。}}",
```

Comment:
From [Format.js docs](https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format):

> one: This category is used for languages that have grammar specialized specifically for one item. Many languages, but not all, use this plural category. **(Many popular Asian languages, such as Chinese and Japanese, do not use this category.)**

So we don't want `one` to be used here, which is why I removed it previously.